.circleci/config.yml

```diff
@@ -270,7 +286,7 @@ jobs:
       - android/save-build-cache:
           cache-prefix: v1a
 
-  assemble-sample-app:
+  assemble-purchase-tester:
```

Comment:
Renamed this job, since it was confusing

Reply:
Is there a reason why this is `assemble` and the other is `build`?

Reply:
Not really. Will rename everything to `assemble` since it's what we're doing in these 👍

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1176/commits/65ab470edf4aba1314a991e529490d2bb80d4be7